views/index.ejs

```diff
@@ -11,8 +11,17 @@
     <strong>Items in cart: <!-- Print the number of items in the cart here --></strong>
     <ul>
       <!-- List the names of the items in your cart here -->
+        <% cart.forEach(function(product) { %>
+        <form action="/cart" method="post">
+          <li><%= product.name %></li>
+          <input type="hidden" name="_method" value="Delete">
+          <input type="hidden" name="id" value="<%= product.id %>">
+          <input type="submit" name="delete" value="Delete">
 
-      <!-- If you're adventurous, try adding a delete form/button for each item to remove it from the cart! -->
+          </form>
+        <% }) %>
+
+              <!-- If you're adventurous, try adding a delete form/button for each item to remove it from the cart! -->
         <!-- Note: if you try the delete form, you'll probably need a line like this inside your form: -->
         <!-- <input type="hidden" name="_method" value="delete"> -->
```

Comment:
Nice job, Chris!! I have very little feedback. Great job watching your indents and spacing to keep things consistent. Makes for some very clean code!